provision/filter_plugins/gcloud_inventory.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env python
+
+from ansible import errors
+try:
+    from configparser import ConfigParser
+except ImportError:
+    from ConfigParser import ConfigParser
+import StringIO
+
+def pop_items(item_list,count):
```

Comment:
I don't see what pop_items() does? Also, I am noting common code in each inventory layout filter. Maybe a library is in order here?